sdk/src/logs/logger.cc

```diff
@@ -33,12 +33,6 @@ const nostd::string_view Logger::GetName() noexcept
 
 nostd::unique_ptr<opentelemetry::logs::LogRecord> Logger::CreateLogRecord() noexcept
 {
-  // If this logger does not have a processor, no need to create a log recordable
-  if (!context_)
```

Comment:
Is context_ always not null?

Reply:
It is never null, a context is always created when building a logger provider.